scripts/build/gen_iter_sections.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Bjarki Arge Andreasen
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Script to generate iterable sections from JSON encoded dictionary containing lists of items.
+"""
+
+import argparse
+import json
```

Comment:
YAML maybe? We seem to converge on YAML as our go-to human-readable serialization format everywhere.

Reply:
Uses an existing JSON artifact, therefore no conversion to YAML.